### DISCUSSION TITLE
Adding an AWS account to an existing Reference Architecture deployment

### DISCUSSION THREAD

**Discussion by jeffreymlewis** [gruntwork-io/knowledge-base#679] [How do I] [answered]
Hello,

I'm adding an AWS account to our existing Reference Arch setup. I found this document very useful.

https://docs.gruntwork.io/guides/reference-architecture/example-usage-guide/adding-a-new-account/intro

However, this doc doesn't explain how to setup domain names for the new account, and it seems the service catalog [route53 module](https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/modules/networking/route53) cannot handle this either. (Validation of the ACM cert fails, I assume because the new domain name is not registered.) KB article https://github.com/gruntwork-io/knowledge-base/discussions/20 indicates that the domain & Route53 need to be setup manually. What are the steps for this?

Thanks,
Jeffrey

---

<ins datetime="2023-03-10T16:21:16Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109973">Tracked in ticket #109973</a></p>
</ins>

**Comment by arsci** [comment, marked as answer]
Hey @jeffreymlewis, there's a few options you can use to setup domain names:

* You can use the [Gruntwork CLI](https://github.com/gruntwork-io/gruntwork#bootstrap-the-domain-names) to register your domain name in the account

* You can configure the domains through through the AWS Console (or AWS CLI)
  * [AWS Documentation: Registering a new domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html)
  * [AWS Documentation: Configuring DNS routing for a new domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-configuring-new-domain.html) (when you register a new domain AWS will automatically configure Route53: creating the hosted zone, creating and configuring the name servers)
